docs/gravity-bridge/gravity-bridge-mainnet-deployment-guide.md

```diff
@@ -1,64 +1,89 @@
 
-  
-# Gravity Bridge Deployment Guide
 
-This guide is intended to assist the community validators with setting up crypto.org Gravity Bridge against `Cronos Testnet 3`  
+# Gravity Bridge Mainnet Deployment Guide
```

Comment:
This document will need to be updated later on. 

This won't work with the current mainet binary

Maybe add some disclosure that this document is for a "future" mainnet

Reply:
@thomas-nguy maybe remove the mainnet guide for now? I don't think it's needed

Reply:
seems reasonable